app/environments/brassbirmingham/brassbirmingham/envs/classes/cards/location_card.py

```diff
@@ -0,0 +1,45 @@
+from consts import *
+from python.print_colors import prCyan, prGreen, prLightGray, prPurple, prRed, prYellow
+
+from .card import Card
+from .enums import CardName, CardType
+
+
+class LocationCard(Card):
+    def __init__(self, name: CardName, isWild=False):
+        super(LocationCard, self).__init__(CardType.location)
+        self.name = name
+        self.isWild = isWild
+
+    def __str__(self) -> str:
+        if self.isWild:
+            return self.name
+
+        if self.name in [STOKE_ON_TRENT, LEEK, STONE, UTTOXETER]:
+            return prCyan(self.name)
+        elif self.name in [BELPER, DERBY]:
+            return prGreen(self.name)
+        elif self.name in [
+            STAFFORD,
+            CANNOCK,
+            WALSALL,
+            BURTON_UPON_TRENT,
+            TAMWORTH,
+        ]:
+            return prRed(self.name)
+        elif self.name in [
+            WOLVERHAMPTON,
+            COALBROOKDALE,
+            DUDLEY,
+            KIDDERMINSTER,
+            WORCESTER,
+        ]:
+            return prYellow(self.name)
+        elif self.name in [NUNEATON, BIRMINGHAM, COVENTRY, REDDITCH]:
+            return prPurple(self.name)
+        elif self.name == BEER1 or self.name == BEER2:
+            return prLightGray(self.name)
```

Comment:
nit: These can be ifs instead of elifs

Reply:
Wouldn't it make more sense for them to be elifs? More than 1 should never run

Reply:
You always return though, so only 1 will ever be run any ways. It's not a huge deal. Convention I've seen usually prefers `if`s over `else if`s, if all the `if`s return a value

Reply:
Not a big deal which is why I noted `nit`. Feel free to resolve